ansible/roles/ceph-grafana/tasks/configure_graphite_web.yml

```diff
@@ -1,7 +1,14 @@
 ---
 - name: Create Graphite DB tables
-  command: /usr/bin/graphite-manage syncdb --noinput
+  command: /usr/bin/graphite-manage migrate --noinput
```

Comment:
Did you test this on clean install? (well, purge + reinstall)

Reply:
I'll do that shortly

Reply:
ok - I'm satisfied.